packages/logging-bunyan/src/index.js

```diff
@@ -133,8 +140,6 @@ LoggingBunyan.prototype.formatEntry_ = function(record) {
     );
   }
 
-  record = extend({}, record);
```

Comment:
Is this record user provided? If so, deleting properties from the original object would probably be considered api breaking.

Reply:
Yes, that definitely would be. The reason I removed this line here is that I added a call to `extend` in the `write` function (line 222 below). So essentially, it gets copied at ingestion into the buffer instead of on the write of the buffer to the API.